source/acquaman/AMScanConfiguration.h

```diff
@@ -168,6 +168,12 @@ class AMScanConfiguration : public AMDbObject
 	virtual QString technique() const;
 
 	/*!
+	  * Returns whether this is an XAS technique scan
+	  */
+
+	bool isXASScan() const;
```

Comment:
Hey @XuetaoLiu-CLS, you could probably remove the "Scan" part from the techniques.  But the other thing is you could make `isXASScan()` virtual, defaulted to false, then have the XAS scan configurations reimplement and return true.  That might be a cleaner solution.